src/lib/dune

```diff
@@ -3,7 +3,7 @@
   (public_name      ftw)
 
   ; ppxs
-  (preprocess (pps ppx_deriving_yojson))
+  (preprocess (pps ppx_deriving_yojson ppx_deriving.enum))
```

Comment:
I don't see why we need `ppx_deriving.enum` (seeing the changes in `backend/types.ml` I'm not convinced it simplifies or helps the code).